spec/index.html

```diff
@@ -2638,14 +2638,15 @@ <h2>Serialization</h2>
     created by representing each <a>quad</a> from the <a>normalized dataset</a>
     in <a>canonical n-quads form</a>,
     sorting them into <a>code point order</a>,
-    and concatenating them.
+    and concatenating them. (Note that each canonical n-quad ends with a new line, 
```

Comment:
```suggestion
    and concatenating them. (Note that each canonical N-Quads statement ends with a new line, 
```
If I am not mistaken, the term 'n-quad' may not be well-defined. It seems a bit more appropriate to use 'statement' instead.

Reply:
The N-Quads grammar defines a [statement](https://w3c.github.io/rdf-n-quads/spec/#grammar-production-statement) which could be referenced. Not a term, but a grammar rule.